post-install.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Removes first line of patternfly.min.css : This assumes this is the line containing the Open Sans font definitions
+tail -n +2 "node_modules/patternfly/dist/css/patternfly.min.css" > "node_modules/patternfly/dist/css/patternfly.fontless.min.css"
```

Comment:
I don't really like this "remove the first line" approach... couldn't a similar `sed` trick as below be done for this step, too?

Reply:
Yeah good point. I've addressed it now